css/main.css

```diff
@@ -0,0 +1,95 @@
+.main-box{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin: auto;
+}
+
+.head-content{
+    margin: auto auto auto auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 60%;
+    margin: auto;
```

Comment:
Margin is used twice here. You could drop one of these. I would suggest dropping the first one.

Reply:
Noted. Made the changes.